### ISSUE TITLE
Can't Mask Bias Weights For Dense Layer

### ISSUE THREAD

**Issue by smgutstein** [jim-meyer/lottery_ticket_pruner#18]
I've noticed that bias weights for a dense layer are not eligible for masking. The reason for this is that in the `_prunables` method, the boolean returned is `(len(weights.shape) > 1 and ...`. 

The bias weights in a dense layer have a shape `(<# of nodes>,)`, which has a length equal to `1`. This causes the boolean to return `False`.

I would have thought that bias weights should also be prunable. Do you purposely want to be certain they're not prunable? If not, are there any "gotcha's" I should be aware of, if I just change the `>` to a `>=` ?

**Comment by jim-meyer** [comment]
AFAIK the papers that this code was inspired from do not talk about pruning bias weights (perhaps because pruning bias weights potentially only saves one add vs pruning other weights potentially saving many multiplies). That said, experimentation is a good thing. :-)
The logic in LotteryTicketPruner._prunable() is:
```
        return len(weights.shape) > 1 and 'keras.layers' in type(layer).__module__ and type(
            layer).__name__ in self.prunable_layer_names
```
Changing this to `... len(weights.shape) >= 1 ...` should do what you want since `iterate_prunables()` should then start including those bias weights in the pruning process.
